### ISSUE TITLE
Assigning nifH cluster I-VI

### ISSUE THREAD

**Issue by NuruddinKhoiry** [moyn413/Singapore-coral-microbes#1]
Hi Ms. Moynihan,

Currently, I am working with the nifH amplicon and I would like to assign the nifH cluster to all ASVs I got. 
I have read your paper and in the supplementary information, it is mentioned: "Cluster assignment done manually by comparing groups of aligned ASVs (≈500 ASVs) with reference sequences [23, 24]". Thus, may I ask you in detail how you assigned the cluster of your ASVs in the file "SGcoral_nifH_dada2_ASV_table_wClusters.tsv"?

Best regards,
Khoiri

**Comment by moyn413** [comment]
Dear Khoiri, 

Thank you for your message. The manual cluster assignment was a bit of a pain. Using the Geneious Prime software, I translated my nifH sequences into protein sequences (with the correct reading frame). Then using the tree from Meheust et al. 2021, along with other nifH sequences from Heller et al 2014, I aligned small groups of my own sequences against these known sequences and clusters. After the protein sequences were aligned, I built phylogenetic trees and used the information from these trees to assign clusters to my own sequences. I did this with small batches of ASVs of my own data against the reference protein sequences. I’m happy to chat more if you want help with this. I’m also working on adding nifH cluster information into the nifH dada2 database, so that this process hopefully can be automated. However, it will probably be another few months before I finish that project.

**Comment by NuruddinKhoiry** [comment]
Dear Ms. Moynihan,

Thank you very much for the explanation. Would you mind sharing with me the protein sequences of the reference tree from Meheust et al. 2021 and other nifH sequences from Heller et al 2014?
"nifH dada2 database with cluster" sounds very interesting. Looking forward to being able to use it in my analysis.

Best regards,
Khoiri

**Comment by moyn413** [comment]
Dear Khoiri, 

I apologize for my very slow response. I've just returned from an extended period of leave. The newick file used to make the tree is on this github page, but it also has all of my own ASVs in it. I've attached here the protein sequences, aligned protein sequences, and tree based off of sequences from Meheust et al. 2021. I hope this is helpful!  
[nifH_tree_ref_protein_sequences.zip](https://github.com/moyn413/Singapore-coral-microbes/files/8119306/nifH_tree_ref_protein_sequences.zip)
